Britannia/localization/english/event_localization/BTWK_culture_events_l_english.yml

```diff
@@ -1,4 +1,20 @@
 ﻿l_english:
- culture_event.002.t:0 "" #Can't think of anything
+ culture_event.002.t:0 "The Frisian language"
  culture_event.002.desc:0 "Language is always changing - and so is the Frisian one. After hundreds of years of changes, the new names of our people and our titles have become unrecognizable compared to earlier forms."
- culture_event.002.a:0 "Interesting..."
\ No newline at end of file
+ culture_event.002.tt:0 "The #color_light_blue Naming Practices#! of the #color_light_blue Frisian#! [culture|E] have updated to #color_goldy_yellow Frisian#!" 
```

Comment:
Better to use the localisation keys of the name lists, e.g. $name_list_frisian$ instead of Frisian

Reply:
Fixed